parl/remote/utils.py

```diff
@@ -191,3 +191,19 @@ def get_subfiles_recursively(folder_path):
             elif len(dirs) == 0:
                 empty_subfolders.append(os.path.normpath(root))
         return python_files, other_files, empty_subfolders
+
+
+def has_module(module_name):
+    '''
```

Comment:
add the comment: 
Check if the python environment has installed the module or package.

Reply:
fixed